mne/io/brainvision/brainvision.py

```diff
@@ -404,6 +462,23 @@ def _check_mrk_version(header):
               'N': 1}  # Newton
 
 
+def _str_to_meas_date(date_str):
```

Comment:
shouldn't this go into `meas_info.py`? maybe we should look what's in there, what the readers need and unify it.

Reply:
this str description of a meas_date is specific to brainvision.

we don't use : datetime.strptime(date_str, '%Y%m%d%H%M%S%f')
elsewhere.